lib/segment/src/segment/entry.rs

```diff
@@ -99,7 +97,7 @@ impl SegmentEntry for Segment {
             hw_counter,
         )?;
 
-        self.process_search_result(internal_results, with_payload, with_vector, hw_counter)
+        self.process_search_result(internal_results, &false.into(), &false.into(), hw_counter)
```

Comment:
It was a parameter before, and it's false now.

Can we use false here, because if a user specified true, it'll be fetched in the separate step that is implemented in this PR?